From a15519db54b0cb197b7cd0c2555274cafac7190a Mon Sep 17 00:00:00 2001
From: Dan Pasanen <invisiblek@cyanogenmod.org>
Date: Thu, 15 Sep 2016 07:29:23 -0500
Subject: [PATCH 6/7] PhoneFactory: fix creating a cdma phone type

* TelephonyComponentFactory.makePhone expects a type of
  PhoneConstants.PHONE_TYPE_GSM or PhoneConstants.PHONE_TYPE_CDMA
  but we're sending in PhoneConstants.PHONE_TYPE_CDMA_LTE which
  is going to end up leaving us with a null Phone object

Change-Id: I76ae6149199e7fa625eb9b85b39d3985e04aecf4
---
 src/java/com/android/internal/telephony/PhoneFactory.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/java/com/android/internal/telephony/PhoneFactory.java b/src/java/com/android/internal/telephony/PhoneFactory.java
index ba777470f..fdbdbef90 100644
--- a/src/java/com/android/internal/telephony/PhoneFactory.java
+++ b/src/java/com/android/internal/telephony/PhoneFactory.java
@@ -195,7 +195,7 @@ public class PhoneFactory {
                     } else if (phoneType == PhoneConstants.PHONE_TYPE_CDMA) {
                         phone = new GsmCdmaPhone(context,
                                 sCommandsInterfaces[i], sPhoneNotifier, i,
-                                PhoneConstants.PHONE_TYPE_CDMA_LTE,
+                                PhoneConstants.PHONE_TYPE_CDMA,
                                 TelephonyComponentFactory.getInstance());
                     }
                     Rlog.i(LOG_TAG, "Creating Phone with type = " + phoneType + " sub = " + i);
-- 
2.17.1

